server/server.go

```diff
@@ -71,7 +71,9 @@ func NewServer(c *Config) (*Server, error) {
 		}
 	case "boltdb":
 		boltOptions := &boltdb.Options{
-			BaseDN:                  s.config.LDAPBaseDN,
+			BaseDN:  s.config.LDAPBaseDN,
+			AdminDN: s.config.LDAPAdminDN,
```

Comment:
I think this should also be done for the Ldif backend to avoid confusion and for runtime compatibility.

Reply:
done